File: lesson_5_2/app/src/main/java/com/kalashnyk/denys/samplemvvmapp/repository/database/pojo/UserResponse.java
package com.kalashnyk.denys.samplemvvmapp.repository.database.pojo;

import com.google.gson.annotations.SerializedName;
import com.kalashnyk.denys.samplemvvmapp.repository.database.entity.UserEntity;

import java.util.List;

public class UserResponse {
    @SerializedName("records")
    private List<UserEntity> mRecords;

    public List<UserEntity> getRecords() {
        return mRecords;
    }

    public void setRecords(List<UserEntity> mRecords) {
        this.mRecords = mRecords;
    }

}

File: lesson_9/app/src/main/java/com/kalashnyk/denys/kotlinbindingsample/repository/database/pojo/UserResponse.kt
package com.kalashnyk.denys.kotlinbindingsample.repository.database.pojo

import com.google.gson.annotations.SerializedName
import com.kalashnyk.denys.kotlinbindingsample.repository.database.entity.UserEntity

class UserResponse {
    @SerializedName("records")
    var records: List<UserEntity>? = null
}